README.md

```diff
@@ -6,6 +6,56 @@
 
 # SW Helpers
 
+## Contents
+
+- [Ways to Use the Libraries](#ways-to-use-the-libraries)
+- [Setting Up](#setting-up)
```

Comment:
Nit: it's common for libraries to use the `Setup` verbiage. Will leave to your discretion.